tests/index.html

```diff
@@ -3,32 +3,32 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Dummy Tests</title>
-    <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Ember Gestures</title>
```

Comment:
still look to be here, I think you are getting tricked by there being two test index.html files.

tests/index.html is the page that runs when you see the tests in a browser.
tests/dummy/app/index.html is the page that acts as the root for the dummy app, which is where this title/meta/bootstrap css should be preserved.